tests/maker.rs

```diff
@@ -121,7 +121,7 @@ impl MakerCli {
         );
 
         // Wait for fidelity bond confirmation
-        await_message(&rx, "Fidelity Transaction");
+        await_message(&rx, "Transaction");
```

Comment:
This can trigger for unrelated logs. It should look for `Transaction confirmed at height`.